### PR TITLE
added support for python3

### PR DESCRIPTION
**Pull request by mukhlesur-millon**
- works on python3 and python2
- extend tests from **Rifat Nabi**'s **jsAvroPhonetic**.
- pylint :10.00/10
- all tests are ok
- remove all deprecated option from `pylint-rc`

> https://github.com/kaustavdm/pyAvroPhonetic/pull/7#issuecomment-230708736

### CODE REVIEW

**Comment by auvipy** [comment]
thanks

File: README.rst [hunk 1]
```diff
@@ -82,7 +78,7 @@ At present, only the text parser has been implemented. It can be accessed as:
 
 ::
 
-    >>> from pyavrophonetic import avro
+    >>> from pyavrophonetic as avro
```
Comment (auvipy) [inline README.rst]: this should be as is

**Comment by auvipy** [comment]
from the don't work on the master branch.

**Comment by mukhlesur-millon** [comment]
updated it.

**Pull request merged**